projects/packages/search/compatibility/unsupported-browsers.php

```diff
@@ -0,0 +1,39 @@
+<?php // phpcs:ignore WordPress.Files.FileName.NotHyphenatedLowercase
+/**
+ * Compatibility for browsers that don't support Instant Search.
+ *
+ * @package automattic/jetpack-search
+ */
+namespace Automattic\Jetpack\Search\Compatibility;
+
+add_filter( 'jetpack_search_classic_search_enabled', __NAMESPACE__ . '\enable_classic_search_for_unsupported_browsers', 10, 1 );
+
+/**
+ * Get the iOS version from the user agent.
+ *
+ * @param string $user_agent The user agent string.
+ * @return null|string The iOS version, or null if not found.
+ */
+function get_ios_version_from_user_agent( $user_agent ) {
+	preg_match( '#\((iPhone|iPad|iPod).*?OS (\d+_?\d?_?\d?).*?\)#', $user_agent, $matches );
+
+	if ( empty( $matches[2] ) ) {
+		return null;
+	}
+
+	$version = str_replace( '_', '.', $matches[2] );
+	return $version;
+}
+
+/**
+ * Force enable Classic Search for browsers for iOS versions < 16.
+ *
+ * @param boolean $classic_search_enabled whether Classic Search is enabled.
+ */
+function enable_classic_search_for_unsupported_browsers( $classic_search_enabled ) {
```

Comment:
I think it might make sense to rename this to better reflect what the code is actually doing. This code doesn't "enable classic search" per say. It only tests if the version is less than 16. Maybe either `is_iOS_less_than_16` or `should_enable_classic_search` or something like that.

Reply:
That's a good point - sorry I only saw this after merging the PR. It probably is not worth a PR itself, and I'll see what I can do in the future. Thanks @a8ck3n